packages/validation-manager/src/BundlerCollectorTracer.ts

```diff
@@ -75,6 +75,10 @@ export interface AccessInfo {
   reads: { [slot: string]: string }
   // count of writes.
   writes: { [slot: string]: number }
+  // slot value, just prior this operation
+  transientReads: { [slot: string]: string }
```

Comment:
we define transient storage as equivalent to storage access. 
why collect them separately?

Reply:
I wonder if it is better to add the entires as "transient" into the same hashes. this will make a simpler impl.
currently, at line 339 there is no check for transient storage. not sure if it is just report bug or actual validation bug.